src/client/Microsoft.Identity.Client.Broker/Microsoft.Identity.Client.Broker.csproj

```diff
@@ -14,10 +14,9 @@
 
   <PropertyGroup Label="NuGet and AssemblyInfo metadata">
     <!--This should be passed from the VSTS build-->
-    <BrokerPackageSemVer Condition="'$(BrokerPackageSemVer)' == ''">0.1.0</BrokerPackageSemVer>
-    <BrokerPackageSemVer>$(BrokerPackageSemVer)</BrokerPackageSemVer>
+    <MsalClientSemVer Condition="'$(MsalClientSemVer)' == ''">4.7.1-preview</MsalClientSemVer>
     <!--This will generate AssemblyVersion, AssemblyFileVersion and AssemblyInformationVersion-->
-    <Version>$(BrokerPackageSemVer)</Version>
+    <Version>$(MsalClientSemVer)-preview</Version>
```

Comment:
If we ever plan to release just the broker what will this version be?

Reply:
We'll have to override to somtheing like `-preview2`

Reply:
What is the expected/common scenario?
1. Msal.Broker always released in sync with Msal main? This is what we do for Msal.Desktop, even if there are no changes to main Msal. 
2.  Msal.Broker will be released out-of-sync with Msal main? In this case it's own version might be better. But then what happens when Msal.Broker goes public - does the version suddenly jump from 0.x.0 to 4.x.x?
3. A hybrid of 1 and 2 above?

Reply:
As discussed, will merge as-is.